Estimator/test/MinLogLHEstimatorTest.cpp

```diff
@@ -22,29 +22,33 @@
 class Gaussian : public ComPWA::Intensity {
 public:
   Gaussian(double mean, double width)
-      : Mean(mean), Width(width), Strength(1.0) {}
+      : Mean(ComPWA::Parameter{"mean", mean}),
+        Width(ComPWA::Parameter{"width", width}),
+        Strength(ComPWA::Parameter{"strength", 1.0}) {}
 
   std::vector<double> evaluate(const std::vector<std::vector<double>> &data) {
     auto const &xvals = data[0];
     std::vector<double> result(xvals.size());
     std::transform(xvals.begin(), xvals.end(), result.begin(), [&](double x) {
-      return Strength *
-             std::exp(-0.5 * std::pow(x - Mean, 2) / std::pow(Width, 2));
+      return Strength.Value * std::exp(-0.5 * std::pow(x - Mean.Value, 2) /
+                                       std::pow(Width.Value, 2));
     });
     return result;
   }
 
   void updateParametersFrom(const std::vector<double> &params) {
-    Mean = params[0];
-    Width = params[1];
-    Strength = params[2];
+    Mean.Value = params[0];
```

Comment:
You might check the length of `params` here.

Reply:
I added a check here. Just a general remark: a simple length check is usually not necessary. I personally would allow for example also smaller `vector<double>`.